Modulo III - POO/C4/EjercicioEnClase1/Telefono.ts

```diff
@@ -0,0 +1,35 @@
+export default class Telefono {
+    protected estaPrendido:boolean;
+    protected bateriaActual: number;
+
+    public constructor (prendido:boolean, carga: number) { 
+
+        this.estaPrendido= prendido;
+        this.bateriaActual= carga;
+        
+    }
+
+
+    public mandarMensaje() {
+        
+    }
+
+    /**
+     * hacerLlamada
+     */
+    public hacerLlamada() {
+        
+    }
+
+    /**
+     * prenderApagar
+     */
+    public prenderApagar() {
+        if (this.estaPrendido) {
+            this.estaPrendido = false
+        }
+        else {
+            this.estaPrendido = true
+        }
```

Comment:
Si bien esto funciona correctamente, se puede resumir negando el estado actual de **estaPrendido** (si el valor es true, la negación es false y viceversa)

`
this.estaPrendido = !this.estaPrendido;
`